src/content/advanced/upgrades/gs-20-release/index.md

```diff
@@ -0,0 +1,92 @@
+---
+linkTitle: Giant Swarm Release v20.0.0
+title: Giant Swarm Release v20.0.0
+description: Giant Swarm release v20 besides many improvements will introduce Kuberntes Release Version 1.25.0. Following handbook should be carefully read by customers upfront the upgrade to prepare the clusters and workloads accordingly.
```

Comment:
Yeah maybe we could use 1.25.x? I'm not sure if we wanna use 1.25.0 because it's already out for a while and has already some Patches, WDYT?